src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -746,5 +751,49 @@ public static ScreenshotEvent onScreenshot(BufferedImage image, File screenshotF
         MinecraftForge.EVENT_BUS.post(event);
         return event;
     }
+    
+    public static List<List<BakedQuad>> sortQuadsByIndex(IBakedModel model)
+    {
+        List<List<BakedQuad>> layers = new ArrayList();
```

Comment:
`ArrayList` is a generic type, you need a type parameter to fully specify the type. `ArrayList` on it's own is a raw type. `ArrayList<String>` for example is not a raw type.